src/lib.rs

```diff
@@ -191,15 +207,20 @@ pub async fn run(config: Config) -> Result<()> {
         let images = rt.podman.as_ref().unwrap().images();
         let mut stream = images.pull(&pull_opts);
 
-        while let Some(pull_result) = stream.next().await {
-            match pull_result {
-                Ok(output) => {
-                    debug!("🔧 {:?}", output);
-                }
-                Err(e) => {
-                    error!("❌ pulling image: {}", e);
+        let present_local = false;
```

Comment:
there's no way to check if the image is present locally for podman? Ideally we would have the same logic for both runtimes, albeit in two separate functions.

Reply:
I added podman support in the most recent push